src/Esfa.Vacancy.Register.Api/Controllers/GlobalErrorController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using System.Web.Http.Description;
+
+namespace Esfa.Vacancy.Register.Api.Controllers
+{
+    public class GlobalErrorController : ApiController
+    {
+        [AllowAnonymous]
+        [Route("api/error")]
+        [ApiExplorerSettings(IgnoreApi = true)]
+        public HttpResponseMessage Get()
+        {
+            try
+            {
+                return Request.CreateErrorResponse(HttpStatusCode.BadRequest, "There was a problem with your request.");
+            }
+            catch {}
```

Comment:
I think it is still a good idea to log in this case. Having empty swallow catch blocks are difficult to understand.

Reply:
Can we put a comment here to explain why we are suppressing errors.